Shift3/dj-starter
# Easy Terraform Provisioning

The idea behind this PR is to make terraform provisioning, and subsequent deployment easier.

## Changes
* Use the `eb-s3-postgres` terraform module which attempts to be a bit more generic than the old supermodule.
* Update the `README` with much easier and more straightforward terraform instructions.
* Added a script to automatically fill out CircleCI with environment vars from the provisioned infrastructure.

## Testing Steps
In order to test this fully, you should try to generate, provision, and deploy your own backend setup.

Requires https://github.com/Shift3/terraform-modules/pull/104 to be merged into `main` before we can merge this.
